src/nvim/main.c

```diff
@@ -834,20 +808,25 @@ void getout(int exitval)
   mch_exit(exitval);
 }
 
-/*
- * Get a (optional) count for a Vim argument.
- */
-static int
-get_number_arg (
-    char_u *p,             /* pointer to argument */
-    int *idx,           /* index in argument, is incremented */
-    int def                    /* default value */
-)
+/// Gets the integer value of a numeric command line argument if given,
+/// such as '-o10'.
+///
+/// @param p pointer to argument
+/// @param idx pointer to index in argument
```

Comment:
I'm a little confused after reading this: http://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdparam

Given that `*idx` is a parameter that is modifed, would the following be satisfactory?
```
/// @param[in] p pointer to argument
/// @param[in,out] idx pointer to index in argument
/// @param[in] def default value
```